src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1513,9 +1512,7 @@ Convolution::ConvolutionExecutor::ConvolutionExecutor(const dnnl::convolution_fo
                                                                 const dnnl::memory::desc& inMemDesc,
                                                                 const dnnl::memory::desc& weightMemDesc,
                                                                 const dnnl::memory::desc& outMemDesc,
-                                                                const dnnl::engine& engine) {
-    execPrim = dnnl::convolution_forward(pd);
-
+                                                                const dnnl::engine& engine) : DnnlExecutor(pd) {
     if (inMemDesc != pd.src_desc()) {
```

Comment:
should we change it to `getSrcDesc()` since `pd.src_desc()` is slow?
also `pd.weights_desc()` and `pd.dst_desc()` in following lines

Reply:
Good point. Changed accordingly. Thank you!